docs/zh/guides/user/managing-knowledge-bases.md

```
@@ -0,0 +1,228 @@
+
+- **已启用**：文档处于活动状态且可搜索
+- **已禁用**：文档已存储但不参与搜索
+- **索引中**：文档正在进行 RAG 处理（如已配置）
+
+## 知识库的 AI 工具
+
+### knowledge_base_search（RAG 模式）
+
+查找相关文档分块的语义搜索工具：
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)